CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Head
+
+* Removed: `stylelint < 4.3.0` compatibility.
+  * Added: `font-family-name-quotes` rule with double quotes where recommended option.
+  * Added: `media-feature-no-missing-punctuation` rule.
```

Comment:
I did it to show that the two new rules are the reason for the removed compatibility, but now I'm wondering whether it's just confusing and unusual within a changelog :) 

I should probably remove the indentation, right?